Changelog.md

```diff
@@ -6,7 +6,7 @@ Language Features:
 Compiler Features:
  * SMTChecker: Support shifts.
  * SMTChecker: Support structs.
- * SMTChecker: Support ``type(T).min`` and ``type(T).max``.
+ * SMTChecker: Support ``type(T).min``, ``type(T).max``, and ``type(I).interfaceId``.
```

Comment:
```suggestion
 * SMTChecker: Support ``type(T).min``, ``type(T).max`` and ``type(I).interfaceId``.
```

Reply:
See oxford comma: https://www.grammarly.com/blog/what-is-the-oxford-comma-and-why-do-people-care-so-much-about-it/